deps/build.jl

```diff
@@ -15,7 +15,8 @@ files = [
   "Incompressible Navier-Stokes"=>"inc_navier_stokes.jl",
   "Stokes equation" => "stokes.jl",
   "Isotropic damage model" => "isotropic_damage.jl",
-  "Fluid-Structure Interaction"=>"fsi_tutorial.jl"]
+  "Fluid-Structure Interaction"=>"fsi_tutorial.jl",
+  "On using DrWatson.jl to authomatise your simulation workflows"=>"validation_DrWatson.jl"]
```

Comment:
I guess that this title will not fit into the left panel of the html page. Can you find a shorter one?

Reply:
I think that authomatise is without h, right?

Reply:
> I guess that this title will not fit into the left panel of the html page. Can you find a shorter one?
Done.
> I think that authomatise is without h, right?
Correct.